### ISSUE TITLE
Fix Slider a11y properties

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#2167]
We currently have `ariaLabelledby` property which is mandatory on `Slider`. Not only does it differ from native prop name, but there are other valid combinations using `aria` attrs which are valid for `Slider`.

https://www.w3.org/TR/wai-aria-practices/#slider_roles_states_props

Actions
- Make `ariaLabelledby` optional and make note of removing it in future releases as native `aria-labelledby` should be used. 
- Are other a11y combinations with `aria-value/min/max/now` upheld?
- Accessible `aria-valuetext` story?

**Comment by oddvernes** [comment]
The aria-labelledby does not seem to work on range-slider (aria-labelledby is added to the wrapper div which has a role of group on the range slider and directly on the input on the simple slider). Tested with mac voiceover and windows narrator.

The {...rest} is added to the wrapper-div in both cases so this will probably need to be changed to the inputs if we want other aria stuff to work.

The ariaLabelledby prop value is used to create ids for the input(s) (which is used for internal sreen-reader only labels) but maybe we can use useId hook for this
